sync/api/projects.py

```diff
@@ -416,6 +417,19 @@ def get_project_submission(project_id: str, submission_id: str) -> Response[dict
     return Response(result=response["result"])
 
 
+def get_submissions(project_id: str) -> Response[dict]:
+    """Get submissions for a project id
+
+    :param project_id: project ID
+    :type project_id: str
+    :return: List of Submission Configuration objects
+    :rtype: dict
+    """
+    recent_submissions = get_default_client().get_project_submissions(project_id)
```

Comment:
might consider page=0&per_page=1 if we support it. that should get you the latest? not sure